docs/reference/glossary.md

```diff
@@ -95,6 +98,10 @@ An **Aptos node** is a peer entity of the Aptos network that tracks the state of
 
 ## B
 
+### Blocks
+
+- Batches of [transactions](#transaction) also called “height” in blockchain literature.
```

Comment:
A block isn't called height?

Block is a group of transactions committed at the same time.  The block number is often referred as "block height" in blockchain literature.  In Aptos, transactions are accessed by ledger version rather than by block.